dashboard/app/controllers/code_reviews_controller.rb

```diff
@@ -6,44 +6,36 @@ class CodeReviewsController < ApplicationController
 
   # GET /code_reviews
   # Returns the list of code reviews and associated comments for the given
-  # project (identified by channel id), script, and level.
+  # project (identified by channel id).
   def index
-    params.require([:channelId, :scriptId, :levelId])
+    params.require([:channelId])
```

Comment:
This means that for a project-backed level progression (where each level in a series uses the same project) all the levels will have the same code review. Is that the intent here?

Reply:
Yes, that was the behavior we settled on after meeting with Mike and Jessie on Monday.